FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -57,13 +57,16 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
     proto.application_info.which_platform_info = FIRSESGetAppleApplicationInfoTag()
     proto.application_info.apple_app_info
       .bundle_short_version = makeProtoString(appInfo.appDisplayVersion)
-    proto.application_info.apple_app_info.network_connection_info
-      .network_type = convertNetworkType(networkType: appInfo.networkInfo.networkType)
-    proto.application_info.apple_app_info.network_connection_info
-      .mobile_subtype = convertMobileSubtype(mobileSubtype: appInfo.networkInfo.mobileSubtype)
     proto.application_info.apple_app_info.os_name = convertOSName(osName: appInfo.osName)
     proto.application_info.apple_app_info.mcc_mnc = makeProtoString(appInfo.mccMNC)
```

Comment:
Should mccMNC be also added as a reserved field? I assume Crashlytics does not capture this information?

Reply:
Yeah that totally makes sense to do 👍

Reply:
Done